examples/pymc3_howto/sampling_conjugate_step.ipynb

```diff
@@ -245,7 +245,7 @@
        "            }\n",
```

Comment:
<p>as we are not storing the summary dataframe anywhere and we only want the rhat, we should use rhat instead. The assertion can be done with:</p><p> </p><p>    assert (az.rhat(trace).to_array() &lt; 1.1).all()</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Unsure. Just re-ran the pre-existing notebooks. haha</p>